pages.th/common/clamdscan.md

```diff
@@ -0,0 +1,36 @@
+# clamdscan
+
+> โปรแกรมแสกนหาไวรัสบนคอมมานด์ไลน์ โดยใช้โปรแกรมพื้นหลัง ClamAV
+> ข้อมูลเพิ่มเติม: <https://www.clamav.net>.
+
+- แสกนข้อบกพร่องของไฟล์หรือไฟล์ในไดเรคทอรี:
+
+`clamdscan {{path/to/file_or_directory}}`
+
+- แสกนข้อมูลจากสตรีมอินพุทมาตรฐาน:
+
+`{{command}} | clamdscan -`
+
+- แสกนไดเรคทอรีปัจจุบันแล้วแสดงผลไฟล์ที่ตรวจพบความบกพร่อง:
+
+`clamdscan --infected`
+
+- ส่งผลการแสกนไปยังไฟล์ที่ระบุ:
+
+`clamdscan --log {{path/to/log_file}}`
```

Comment:
pla
Please translate the placeholders, like `path/to/quarantine_directory`, `path/to/log_file`, etc., for get this easier for non-English speakers.

Reply:
done